tamr_toolbox/low_latency/llm.py

```diff
@@ -0,0 +1,181 @@
+from typing import List, Dict, Optional, Literal
+import json
+from collections import defaultdict
+import logging
+import time
+
+from tamr_unify_client import Client
+from tamr_unify_client.operation import Operation
+
+from tamr_toolbox.models.data_type import JsonDict
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def update_llm_data(
+    tamr_client: Client,
+    *,
+    project_name: str,
+    do_update_clusters: bool = True,
+    do_use_manual_clustering: bool = False,
+) -> None:
+    """
```

Comment:
Are these comments?

Reply:
These are the function arguments with typehints (indicators of datatype; not enforced by Python, but they help documentation/readability)

Reply:
yes these are docstrings https://www.programiz.com/python-programming/docstrings